dotCMS/dependencies.gradle

```diff
@@ -39,11 +39,12 @@ dependencies {
         exclude(group: 'org.apache.logging.log4j')
     }
 
-    compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.8.2'
-    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.8.2'
-    compile group: 'org.apache.logging.log4j', name: 'log4j-1.2-api', version: '2.8.2'
-    compile group: 'org.apache.logging.log4j', name: 'log4j-jcl', version: '2.8.2'
-    compile group: 'org.apache.logging.log4j', name: 'log4j-web', version: '2.8.2'
+    compile group: 'com.lmax', name: 'disruptor', version: '3.3.4'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.10.0'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.10.0'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-1.2-api', version: '2.10.0'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-jcl', version: '2.10.0'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-web', version: '2.10.0'
```

Comment:
we shouldn't upgrade log4j until proguard supports java 9. The disruptor jar should be downgraded instead